### ISSUE TITLE
Indication of model authors

### ISSUE THREAD

**Issue by draeger** [opencobra/cobrapy#907]
The SBML format allows authors to embed their contact information in the form of a VCard, including given name(s) and surname(s), organization, and e-mail address. Also, there are fields for creation date and modification date (see, for instance, [SBML Level 3 Version 1](https://doi.org/10.1515/jib-2017-0080) section 6.7, p. 97). This information can be linked to any model component in a history object. In this way, individual parts of models can be associated with specific authors.

Are there any plans to support this as a feature in COBRApy? Being able to attach a VCard just to the model object itself could certainly be helpful to indicate so the authors of a model directly.

**Comment by Midnighter** [comment]
Definitely would like to support this.

**Comment by ensakz** [comment]
People usually name GEM by the initials using model.name cobrapy function

**Comment by matthiaskoenig** [comment]
Hi @draeger, 
sorry for the slow reply. Currently the SBML parser is already supporting the feature in a beta version. It is basically using the SBML ModelHistory element and parses it into a python dictionary which can be accessed on the cobra model. This is not really documented but already possible. On SBML export this is not working yet, i.e. the dictionary is not yet serialized in the SBML.

So basically one would just add the typical model author information on the model (name, organization, email) which afterwards would be written in the ModelHistory with the creation date.

I will fix this as part of my next round of SBML bugfixes for cobrapy (busy times right now, but I will likely find time in the next few days).

Best Matthias

**Comment by cdiener** [comment]
Now tracked in #1237.
